### PR TITLE
Create routes for separate APIs

### PR DESCRIPTION
**Pull request by ChristinaLisa**
break out of review endpoints and authentication

### CODE REVIEW

File: backend/models/remedies.js
Comment (alj-b) [inline backend/models/remedies.js:1]: So the remedies model is specifically used for the rating then?

File: backend/app.js [hunk 1]
```diff
@@ -81,7 +65,7 @@ app.get('/getRemedyRecommendation', catchAsynch(async (req, res) => {
         },
         data:
         {
-            "symptoms": body
+            "symptomsMatched": body
```
Comment (rjeantet) [inline backend/app.js:68]: Here DS expects symptoms

**Comment by alj-b** [comment]
@rjeantet 
I whitelisted 0.0.0.0/0 (or something like that) in Mongo so IP addresses shouldn't be the issue either :thinking:

**Comment by rjeantet** [comment]
@ChristinaLisa this is outdated? Close?

**Pull request merged**